lib/src/track/web/_audio_html.dart

```diff
@@ -27,6 +30,15 @@ void startAudio(String id, rtc.MediaStreamTrack track) {
   final audioStream = html.MediaStream();
   audioStream.addTrack(track.jsTrack);
   audioElement.srcObject = audioStream;
+  return audioElement.play();
+}
+
+Future<void> startAllAudioElement() async {
+  for (final element in _audioElements.values) {
+    if (element is html.AudioElement) {
+      await element.play();
```

Comment:
in order to work around [this issue](https://stackoverflow.com/questions/9811429/html5-audio-tag-on-safari-has-a-delay/54119854#54119854)
the js client looks at the `state` of a dummy `AudioContext`. 
I guess you could leave out all the `expWebAudioMix` handling for flutter and just use `getNewAudioContext().state === 'running';`

Reply:
yeah, playback delay can be removed after adding AudioContext
https://github.com/livekit/client-sdk-flutter/pull/285/commits/97ce3114fbb1f9bd3058a819c66f587e38f2dfa3